Fix for Issue #110

Alexyoe opened the pull request:
Fix for issue #110.

- Replaced `requestAnimationFrame()` with a custom function that doesn't sleep.
- Added FPS selector to style section of config menu.

**More info**: Issue is related to electron/electron#9567. The function `requestAnimationFrame()` sleeps when not in focus causing video playback to freeze when more than one monitor is present. I replaced this function with a custom one that runs regardless of focus. I added the FPS number input to settings because the original `requestAnimationFrame()` fires based on monitor refresh rate so an option is needed to set this if your monitor is not 60hz.

OrangeJedi commented:
Wow! Thanks for going through andďnot only finding the problem, but writing the code to fix it! I don't think I would have been able to fix this myself. 

I'm assuming that you tested the new code and it fixes #110 without any side effects. If everything else checks out can add it to the next release. However, I'm thinking that it might be a good idea to add another setting next to the FPS input to toggle between the custom function and `requestAnimationFrame()` with the old method as the default. Most users don't seem to be having this issue and having to manually set your FPS out of the box doesn't seem like lots of fun. Do you think you could add the toggle?

Alexyoe commented:
No problem! I'm just glad to help! I'll go ahead and add an option to enable and disable the alternate rendering method and submit a new pull request.

Alexyoe commented:
Sorry for the delay on this. It's been a busy few weeks for me. I've updated my fork with a toggle for the Alternate Render Method and merged your recent commits.

Changes made:

- Moved Alternate Render Method settings to Multiscreen section.
- Added toggle for Alternate Render Method.
- Added function to show/hide fps setting if Alternate Render Method is toggled
- Added switch statement to `createSSWindow()` as after Aerial is built `"argv"` is undefined and throws an error.
